packages/kbn-optimizer/src/optimizer/get_mtimes.ts

```diff
@@ -17,12 +17,12 @@
  * under the License.
  */
 
-import Fs from 'fs';
+import fs from 'fs';
 
 import * as Rx from 'rxjs';
 import { mergeMap, toArray, map, catchError } from 'rxjs/operators';
 
-const stat$ = Rx.bindNodeCallback(Fs.stat);
+const stat$ = Rx.bindNodeCallback(fs.stat);
```

Comment:
I think we should fix the type signature here so that `Rx.bindNodeCallback()` doesn't infer the incorrect type, a pattern that I think should be used elsewhere too rather than just overriding `unknown` types later on.

```suggestion
const stat$ = Rx.bindNodeCallback<fs.PathLike, fs.Stats>(fs.stat);
```

![image](https://user-images.githubusercontent.com/1329312/78831088-e9596e00-799d-11ea-8952-6a6212b79ac5.png)